### ISSUE TITLE
Treeview is not accessible using JAWS 2020

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6665]
The is an issue collecting known problems using the JAWS screen reader on the experimental Treeview component.

## Environment

JAWS 2020 on Firefox latest in Windows 10

## Issues:

- [ ] The user is only told their place in the list when the list receives initial focus, but not as they're navigating through the list. 
  _Expectation here is something closer to the functionality seen in a multiselect where the user is updated on their position as they navigate_

- [ ] Hitting enter or space bar selects the focused list item and does not toggle open/close when nested lists are present
  _The expected on click event for a tree view node with a sub-node is to toggle open/close state on `enter` or `space`_

- [ ] The reader does not describe list items with nested lists as opened or closed when they receive focus. 
  _Expected behavior is for a list item with nested children to always describe it's current state: open or closed_

- [ ] When the Treeview receives focus the entire component (lists and all child lists) are read in their entirety
  _The Treeview label should be read, and individual list items should then be read as they receive focus_

**Comment by dakahn** [comment]
closing in favor of #6792
